app/controllers/import_controller.rb

```diff
@@ -0,0 +1,9 @@
+class ImportController < ApplicationController
+
+  def contact
```

Comment:
Why are you still using this action called `contact`? The right way to list records is `index`.